src/module/linux/netlink.c

```diff
@@ -0,0 +1,41 @@
+#include "netlink.h"
+#include <linux/netlink.h>
+#include <net/netlink.h>
+#include <net/net_namespace.h>
+
+char krf_log_msg_buf[KRF_NETLINK_BUF_SIZE];
```

Comment:
I just realized this'll probably do the wrong thing when multiple tasks/CPUs are executing syscalls at once. I'm switching this to a per-CPU variable locally.